hcmanager/hcmanager_test_data.go

```diff
@@ -60,12 +60,16 @@ func getNamespace() string {
 	return "metal3"
 }
 
+func getTestProfileName() string {
+	return "testProfileName"
+}
+
```

Comment:
Something I did not realize at first, the naming of this file is incorrect. If you are using it for test only, it should end in _test.go . so you need to change it to hcmanager_data_test.go

Reply:
@maelk Done